CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v0.2.0 - 2020-08-19
+
+### Features
+- Add support for inversion of intervals (by @henryksloan)
+- Add support for descending scales (by @henryksloan)
```

Comment:
Looks good, maybe we could mention `second_note_down_from`. Maybe something like "Add support for descending a note by an interval".